omnigibson/systems/macro_particle_system.py

```diff
@@ -61,6 +60,9 @@ def initialize(cls, simulator):
         particle_template = cls._create_particle_template()
         simulator.import_object(obj=particle_template, register=False, auto_initialize=True)
 
+        assert len(particle_template.links) == 1, "MacroParticleSystem particle template has more than one link"
```

Comment:
can we add a comment here as to why these asserts are needed? I'm guessing it's because we want to make sure the template mesh has no ambiguity, i.e.: (a) not articulated, and (b) only a single visual mesh that can be used

Reply:
done